app/Enums/TransactionTypeEnum.php

```diff
@@ -12,13 +12,5 @@ final class TransactionTypeEnum
 
     public const VOTE = 3;
```

Comment:
types like these no longer exist as numeric transaction types are a thing of the past. Can we remove this completely?